lib/features/digital_guide_view/tabs/rooms/data/repository/rooms_repository.dart

```diff
@@ -3,24 +3,28 @@ import "package:flutter_riverpod/flutter_riverpod.dart";
 import "package:riverpod_annotation/riverpod_annotation.dart";
 
 import "../../../../data/api/digital_guide_get_and_cache.dart";
-import "../../../../data/models/digital_guide_response.dart";
+import "../../../../data/models/level.dart";
+import "../../../../data/models/level_with_regions.dart";
+import "../../../../data/models/region.dart";
 import "../models/digital_guide_room.dart";
 
 part "rooms_repository.g.dart";
 
 @riverpod
 Future<IList<DigitalGuideRoom>> roomsRepository(
   Ref ref,
-  DigitalGuideResponse building,
+  LevelWithRegions level,
 ) async {
-  final String endpoint = "rooms/?building=${building.id}";
+  Future<DigitalGuideRoom> getDigitalGuideRoom(int roomId) async {
+    final String endpoint = "rooms/$roomId";
```

Comment:
[nit] do as you like, but letting you know that this `String` is not needed, as the types are inferred in such case

```suggestion
    final endpoint = "rooms/$roomId";
```

Reply:
Thank you! I will apply changes, padding are to big rn, I agree